sql/commands/sql-create-connection.mdx

```diff
@@ -1,44 +1,103 @@
-| service.name        | The service name of the endpoint service.                                                                                        |
-| tags                | Optional. The AWS tags used to check for resource leakage. This parameter should have the format: key1=value1, key2=value2, .... |
+| `type` | Required. The type of connection. Supported values: `kafka`, `schema_registry`. |
+| `properties.bootstrap.server` | The Kafka bootstrap server addresses. Required when `type` is `kafka`. |
+
+
+<Accordion title="Click to see all supported properties for Kafka connection.">
+
+The following properties are optional and can be included in the Kafka connection configuration as needed:
+
+ **SSL/SASL authentication:**
+  - `properties.security.protocol`
+  - `properties.ssl.endpoint.identification.algorithm`
+  - `properties.ssl.ca.location`
+  - `properties.ssl.ca.pem`
+  - `properties.ssl.certificate.location`
+  - `properties.ssl.certificate.pem`
+  - `properties.ssl.key.location`
+  - `properties.ssl.key.pem`
+  - `properties.ssl.key.password`
+  - `properties.sasl.mechanism`
+  - `properties.sasl.username`
+  - `properties.sasl.password`
+  - `properties.sasl.kerberos.service.name`
+  - `properties.sasl.kerberos.keytab`
+  - `properties.sasl.kerberos.principal`
+  - `properties.sasl.kerberos.kinit.cmd`
+  - `properties.sasl.kerberos.min.time.before.relogin`
+  - `properties.sasl.oauthbearer.config`
+
+ **PrivateLink connection:**
+  - `privatelink.targets`
+  - `privatelink.endpoint`
+
+ **AWS authentication:**
+  - `aws.region`
+  - `endpoint`
+  - `aws.credentials.access_key_id`
+  - `aws.credentials.secret_access_key`
+  - `aws.credentials.session_token`
+  - `aws.credentials.role.arn`
+  - `aws.credentials.role.external_id`
+</Accordion>
 
-<Note>
-You can either tag the VPC endpoints by specifying the `tags` parameter when using the `CREATE CONNECTION` command or by specifying the environment variable `RW_PRIVATELINK_ENDPOINT_DEFAULT_TAGS`. When specifying the tags, follow the format of `key1=value1, key2=value2, ...`. If both are specified, the tags specified in the environment variable will be appended to the ones specified by the `tags` parameter.
-</Note>
 
 ## Example
 
-The statement below creates an AWS PrivateLink connection.
+To connect to a schema registry:
 
 ```sql
-CREATE CONNECTION connection_name WITH (
-    type = 'privatelink',
-    provider = 'aws',
-    service.name = 'com.amazonaws.xyz.us-east-1.abc-xyz-0000'
+CREATE CONNECTION sr_conn WITH (
+  type = 'schema_registry',
+  schema.registry = 'http://...',
+  schema.registry.username = 'admin_user',
+  schema.registry.password = 'schema_registry_password'
+);
+```
+
+To create a Kafka connection that securely integrates secrets:
+
+```sql
+CREATE CONNECTION conn_kafka WITH (
+    type = 'kafka',
+    properties.bootstrap.server = SECRET sec_broker,
```

Comment:
making properties.bootstrap.server a secret is not a common practice. Let's refer to a sasl one

```
   connector='kafka',
   properties.bootstrap.server='<broker addr>',
   properties.sasl.mechanism='PLAIN',
   properties.security.protocol='SASL_PLAINTEXT',
   properties.sasl.username=SECRET <username>,
   properties.sasl.password=SECRET <password>
```